projects/packages/patchwork-redefine-exit/tests/php/fixtures/die.php

```diff
@@ -4,5 +4,6 @@
 	// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
 	die( $arg );
 } else {
+	// @phan-suppress-current-line UnusedPluginSuppression @phan-suppress-next-line PhanParamTooFewInternal -- Phan bug with PHP 8.4: https://github.com/phan/phan/issues/4888
```

Comment:
I wonder why you went with suppression here instead of adding a zero as with everywhere else?

Reply:
In this case it's a test specifically for the no-arg variant of `die()` (a valid variant despite what Phan claims on PHP 8.4), so adding an arg defaults the point of that test.